bot/exts/evergreen/splitify.py

```diff
@@ -0,0 +1,130 @@
+import math
+import random
+from io import BytesIO
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+
+MAX_SQUARES = 193_600
```

Comment:
Any specific reason for why you are limiting the squares to this particular number?

Reply:
A greater number than 193,600 causes me PIL errors, I did a lot of testing. If I wanted, I could probably recode it to work with bigger numbers, but since 193,000 squares (split pieces) is already so many for a single Discord PFP, I decided it would serve as a great cap.

Reply:
Ah ok, thanks!